src/_gettsim/synthetic.py

```diff
@@ -157,12 +157,41 @@ def create_basic_households(
     ]
     group_ids = [f"{g}_id" for g in exogenous_groupings]
     df["p_id"] = df.index
+
+    # Create Elternteil IDs
```

Comment:
We'll probably want to change these eventually. Would be better to do it directly rather than setting things up old-style and then doing research. However, this whole function can be updated once the new structure is used everywhere.